src/Controller/Front/AnimalController.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace App\Controller\Front;
+
+use App\Entity\Animal;
+use App\Form\AnimalType;
+use App\Repository\AnimalRepository;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+
+use Symfony\Component\HttpKernel\Exception\AccessDeniedHttpException;
+
+/**
+ * @Route("/animal", name="animal_")
+ */
+class AnimalController extends AbstractController
+{
+    /**
+     * @Route("/{id}/show", name="show", methods={"GET"})
+     */
+    public function show_animal(Animal $animal, AnimalRepository $animalRepository): Response
```

Comment:
https://symfony.com/doc/current/contributing/code/standards.html#naming-conventions
```suggestion
    public function showAnimal(Animal $animal, AnimalRepository $animalRepository): Response
```